CASC/PortfolioMode.cpp

```diff
@@ -456,76 +469,60 @@ bool PortfolioMode::runSchedule(Schedule& schedule)
 }
 
 /**
- * Return the intended slice time in deciseconds and assign the slice
- * vstring with chopped time limit to @b chopped.
+ * Return the intended slice time in deciseconds
  */
-unsigned PortfolioMode::getSliceTime(vstring sliceCode,vstring& chopped)
+unsigned PortfolioMode::getSliceTime(const vstring &sliceCode)
 {
   CALL("PortfolioMode::getSliceTime");
 
   unsigned pos = sliceCode.find_last_of('_');
   vstring sliceTimeStr = sliceCode.substr(pos+1);
-  chopped.assign(sliceCode.substr(0,pos));
   unsigned sliceTime;
   ALWAYS(Int::stringToUnsignedInt(sliceTimeStr,sliceTime));
   ASS_G(sliceTime,0); //strategies with zero time don't make sense
 
   unsigned time = _slowness * sliceTime + 1;
+  // TODO: huh?
```

Comment:
I have no idea what's going on here. If it's less than 10 deciseconds, increment by one?